README.md

```diff
@@ -72,7 +72,7 @@ this reason, only the latest Hubble CLI version is maintained.
 
 | Version                                              | Release Date         | Maintained | Supported Cilium Version | Artifacts                                                               |
 |------------------------------------------------------|----------------------|------------|--------------------------|-------------------------------------------------------------------------|
-| [v0.12](https://github.com/cilium/hubble/tree/v0.12) | 2023-10-12 (v0.12.2) | Yes        | Cilium 1.14 and older    | [GitHub Release](https://github.com/cilium/hubble/releases/tag/v0.12.2) |
+| [v0.12](https://github.com/cilium/hubble/tree/v0.12) | 2023-12-08 (v0.12.3) | Yes        | Cilium 1.14 and older    | [GitHub Release](https://github.com/cilium/hubble/releases/tag/v0.12.3) |
 | [v0.11](https://github.com/cilium/hubble/tree/v0.11) | 2023-06-07 (v0.11.6) | No         | Cilium 1.13 and older    | [GitHub Release](https://github.com/cilium/hubble/releases/tag/v0.11.6) |
 | [v0.10](https://github.com/cilium/hubble/tree/v0.10) | 2022-06-22 (v0.10.0) | No         | Cilium 1.12 and older    | [GitHub Release](https://github.com/cilium/hubble/releases/tag/v0.10.0) |
 | [v0.9](https://github.com/cilium/hubble/tree/v0.9)   | 2021-11-30 (v0.9.0)  | No         | Cilium 1.11 and older    | [GitHub Release](https://github.com/cilium/hubble/releases/tag/v0.9.0)  |
```

Comment:
Please let's remove v0.9 on the way as Cilium v1.11 is not supported anymore.
```suggestion
```

Reply:
done